File: 119. Pascal's Triangle II/C++/Pascal's Triangle II.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        vector<int> result(rowIndex + 1, 0);
        result[0] = 1;
        for (int row = 1; row <= rowIndex; row++) {
            for (int i = row; i > 0; i--) {
                result[i] += result[i - 1];
            }
        }
        return result;
    }
};

File: 119. Pascal's Triangle II/Python/Pascal's Triangle II.py
class Solution(object):
    def getRow(self, rowIndex):
        """
        :type rowIndex: int
        :rtype: List[int]
        """
        result = [0] * (rowIndex + 1)
        result[0] = 1
        for row in range(1, rowIndex + 1):
            for i in range(row, 0, -1):
                result[i] += result[i - 1]
        return result
